stryker/guides/stryker-diff-runner.md

```diff
@@ -0,0 +1,36 @@
+# stryker-diff-runner
+`stryker-diff-runner` is an utility script that allow you to run mutation testing on the part of the code that have diff with the `origin/master` branch.
+
+To get started using `stryker-diff-runner`, please follow these steps:
+1. Install required packages for the script to run: `npm i -D @stryker-mutator/core @stryker-mutator/api stryker-diff-runner`
+2. Install required packages for the targeted project typology (Angular, React.js, Vue.js or Node.js).
+3. Add at least the `mutate` section of the *stryker.conf.js* file with the files you want to have mutation on.
```

Comment:
I'm missing the step 4: how to run it. I assume something like `npx stryker-diff-runner`?